lib/internal/streams/lazy_transform.js

```diff
@@ -22,7 +25,7 @@ function makeGetter(name) {
     this._writableState.decodeStrings = false;
 
     if (!this._options || !this._options.defaultEncoding) {
-      this._writableState.defaultEncoding = crypto.DEFAULT_ENCODING;
+      this._writableState.defaultEncoding = getDefaultEncoding();
```

Comment:
Is this intentional, or a left over from a different PR?

Reply:
See the OP:

> The first patch comes from #24396

Reply:
:man_facepalming: I was looking for the reference in the OP, but completely missed the first line.
Sorry.